src/components/themed/TransactionListTop.tsx

```diff
@@ -421,31 +420,32 @@ export class TransactionListTopComponent extends React.PureComponent<Props, Stat
       <>
         <View style={styles.balanceBoxWalletNameCurrencyContainer}>
           <EdgeTouchableOpacity accessible={false} style={styles.balanceBoxWalletNameContainer} onPress={this.handleOpenWalletListModal}>
-            <CryptoIconUi4 sizeRem={1.5} tokenId={tokenId} walletId={wallet.id} />
             <EdgeText accessible style={styles.balanceBoxWalletName}>
               {walletName}
             </EdgeText>
-            <Ionicons name="chevron-forward" size={theme.rem(1.5)} color={theme.iconTappable} />
+            <View style={styles.balanceBoxChevronAdjust}>
```

Comment:
remove the View and put the style on the `Ionicons`

Reply:
Thought it didn't take styles for some reason... Removed View